README.md

```diff
@@ -1 +1,20 @@
-# libsat355
\ No newline at end of file
+# `libsat355`
+Libsat355 is a library encompassing all Zeptomoby orbital calculation code, which is then utilized in Mobilesat. </br>
+Libsat355 is multi-platform compatible, for Windows, iOS, and MacOS. While the library is written using C++, its C briding header allows the program to work on multiple platforms. </br>
+It utilizes NVI archetecture, which allows the implementation to be updated without having to change the interface. </br>
+This functionality has already been used to allow the implementation to utilize multithreading. </br>
+The profiler indicates that the majority of the time is currently spent sorting the TLE data. </br>
+As this time is mainly spent in the quicksort method, not much optimization is needed, other than possibly finding a more specialized sorting method. </br>
+
+## Components:
+Library
++   DLL (Windows)
++   Dylib (Mac)
++   Static Library (iOS)
+
+Unit Tests
++ test1.cpp
++ StarlinkTLE.txt
+
+Build Instructions
++ Invoke the following CMake command: `cmake --build ./build`
```

Comment:
you should add a multi line triple-backtick section to describe the build procedures.
```
cd ./someplace
cmake ...etc.
```

also you need to say what the difference is between Mac and Ios builds. Meaning: how to invoke a Mac instead of Ios build and vice-versa.

Also also: add a section stating the buildtools requirements...
you know: Visual Studio? Visual Studio Code? which version or better. Same with Xcode?